docs/source/examples/checks/methodology/single_feature_contribution_train_test.ipynb

```diff
@@ -5,123 +5,83 @@
    "id": "21c5d5bc",
```

Comment:
<p>The PPS calculation is <em>done</em> using the...</p><p> </p><p>Also, think we need also to explain what pps is: pps is calculated by training a model to predict the label from the single feature, and measuring it's score compared to the best score possible (result=1) and the baseline provided by a constant prediction (result = 0). Can look in the package for a better explanation perhaps. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>